addons/nuxeo-core-binarymanager-cloud/nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/core/storage/sql/S3BinaryManager.java

```diff
@@ -162,6 +155,8 @@ protected String getDefaultDigestAlgorithm() {
 
     public static final String DELIMITER = "/";
 
+    /** @deprecated since 11.1, now unused */
+    @Deprecated
     private static final Pattern MD5_RE = Pattern.compile("[0-9a-f]{32}");
```

Comment:
Maybe this **private** constant can be removed is the method **isMD5** is rewritten (cf comment below)

Reply:
See above, actually not with the way the code is written now.